source/glossary.rst

```diff
@@ -144,6 +144,27 @@ and another geared toward developers. You can find the
    qzv
      See :term:`visualization`.
 
+   rarefy
+     A technique to minimize uneven sampling depth by subsampling without 
+     replacement so that all :term: `samples <sample>` have the same sampling
+     depth. As decribed in `Waste not, want not: 
+     Revisiting the analysis that called into question the practice of 
+     rarefaction
+     <https://www.biorxiv.org/content/10.1101/2023.06.23.546312v1.full>` by
+     Patrick Schloss.
```

Comment:
```suggestion
     depth. See `Schloss (2024) <https://doi.org/10.1128/msphere.00355-23>` for additional information. 
```

Shortening this for readability, and also citing the final published paper instead of the preprint.